src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,59 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Lotto {
+    private final List<LottoNumber> numbers;
+
+    public Lotto(String[] numbers) {
+        this(StringToList(numbers));
+    }
+
+    public Lotto(List<LottoNumber> numbers) {
+        this.numbers = numbers;
+    }
+
+    private static List<LottoNumber> StringToList(String[] numbers) {
+        List<LottoNumber> list = new ArrayList<>();
+        for (int i = 0; i < numbers.length; i++) {
+            list.add(new LottoNumber(Integer.parseInt(numbers[i])));
```

Comment:
```suggestion
            list.add(new LottoNumber(numbers[i]));
```
LottoNumber에 String을 인자로 받는 생성자를 추가해 보면 어떨까?